client.go

```
@@ -149,7 +210,7 @@ func retryBackoff(retry int, minBackoff, maxBackoff time.Duration) time.Duration
 		return maxBackoff
 	}
 
-	d = minBackoff + time.Duration(rand.Int63n(int64(d)))
+	d = minBackoff + time.Duration(rand.Int64N(int64(d)))
```

golangci-lint (2.12.2)

[high] G404: Use of weak random number generator (math/rand or math/rand/v2 instead of crypto/rand)

(gosec)